packages/account/src/Components/file-uploader-container/file-uploader-container.tsx

```diff
@@ -1,23 +1,81 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+//@ts-nocheck [TODO] - Need to fix typescript errors in this file
 import React from 'react';
```

Comment:
Should we destructure { useEffect, Fragment } here? 🤔

Reply:
No. This component is used in CFD as well. It breaks there